diff --git a/compiler/rustc_apfloat/src/lib.rs b/compiler/rustc_apfloat/src/lib.rs
index 4a845fcb691..6e7b2c6fc46 100644
--- a/compiler/rustc_apfloat/src/lib.rs
+++ b/compiler/rustc_apfloat/src/lib.rs
@@ -102,7 +102,6 @@ macro_rules! unpack {
 /// Category of internally-represented number.
 #[derive(Copy, Clone, PartialEq, Eq, Debug)]
 pub enum Category {
-    Infinity,
     NaN,
     Normal,
     Zero,
